pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -161,7 +161,7 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		var ok bool
 		newRc, ok = obj.(*api.ReplicationController)
 		if !ok {
-			return cmdutil.UsageError(cmd, "%s does not specify a valid ReplicationController", filename)
+			return cmdutil.UsageError(cmd, "%s does not specify a valid ReplicationController (%#v)", filename, obj)
```

Comment:
Latest isn't safe from here - the object could not be supported by the scheme.  Obj is always going to be huge.  Can we just omit it?

> On May 13, 2015, at 9:32 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/kubectl/cmd/rollingupdate.go:
> 
> > @@ -161,7 +161,7 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
> >  		var ok bool
> >  		newRc, ok = obj.(*api.ReplicationController)
> >  		if !ok {
> > -			return cmdutil.UsageError(cmd, "%s does not specify a valid ReplicationController", filename)
> > +			return cmdutil.UsageError(cmd, "%s does not specify a valid ReplicationController (%#v)", filename, obj)
> That serialization tends to look pretty ugly and it's not what a user is accustomed to seeing. Perhaps attempting a json encoding using latest?
> 
> —
> Reply to this email directly or view it on GitHub.
>